### PR TITLE
Fixup of #12861: custom annotations in Word break annotations in elem…

### PR DESCRIPTION
**Pull request by LeonarddeR**
### Link to issue number:
Fixup for #12861

### Summary of the issue:
The following error is in the log when vieiwing annotations in the elements list:

```
ERROR - queueHandler.flushQueue (19:47:20.811) - MainThread (9476):
Error in func ElementsListDialog.initElementType
Traceback (most recent call last):
  File "queueHandler.pyc", line 55, in flushQueue
  File "browseMode.pyc", line 1054, in initElementType
  File "browseMode.pyc", line 1074, in filter
  File "NVDAObjects\UIA\wordDocument.pyc", line 139, in label
  File "NVDAObjects\UIA\wordDocument.pyc", line 95, in getCommentInfoFromPosition
AttributeError: 'WordBrowseModeDocument' object has no attribute '_UIACustomAnnotationTypes'
```

### Description of how this pull request fixes the issue:
Hopefully fix this, though I don't have a way to reproduce how the custom annotations work. @michaelDCurran  could you please take a look?

### Testing strategy:
To be tested with a real test case, to be provided.

### Known issues with pull request:
None known

### Change log entries:
None needed.

### Code Review Checklist:

- [ ] Pull Request description:
  - description is up to date
  - change log entries
- [ ] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [ ] API is compatible with existing add-ons.
- [ ] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [ ] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English

### CODE REVIEW

**Comment by michaelDCurran** [comment]
Is there an issue or exact steps to reproduce for this?

**Comment by leonardder** [comment]
Steps are as follows:

1. Open Word
2. Type some text
3. Add a comment with ctrl+alt+m
4. Close the comment pane with escape
5. Activate browse mode
6. Press NVDA+F7 to open the elements list
7. Activate annotations

The mentioned error will be in the log.

File: source/NVDAObjects/UIA/wordDocument.py [hunk 1]
```diff
@@ -101,26 +102,24 @@ def getCommentInfoFromPosition(position):
...
-			comment = obj.makeTextInfo(textInfos.POSITION_ALL).text
-			tempObj = obj.previous.previous
-			authorObj = tempObj or obj.previous
-			author = authorObj.name
-			if not tempObj:
-				return dict(comment=comment, author=author)
-			dateObj = obj.previous
-			date = dateObj.name
-			return dict(comment=comment, author=author, date=date)
+			return dict(comment=f"kaas {comment}", author=author, date=date)
```
[the remark this answers is not in the document]
Comment (leonardder) [reply source/NVDAObjects/UIA/wordDocument.py:105]: That means cheese 🤣

```suggestion
			return dict(comment=comment, author=author, date=date)
```

**Comment by lukaszgo1** [comment]
Given that this PR fixes regression introduced during  2022.1 development cycle it seems logical to at least try to get it into 2022.1 if possible. cc @michaelDCurran

**Comment by michaelDCurran** [comment]
Superseded by pr #13341
Support for custom comment annotation types was removed from getCommentInfoFromPosition as it is not needed.

**Pull request closed without merging**